build.gradle

```diff
@@ -30,10 +30,12 @@ configurations {
 }
 
 dependencies {
-    api "org.grails:grails-core:$grailsVersion"
-    api "org.gebish:geb-spock:$gebSpock", {
-        exclude group: 'org.codehaus.groovy', module: 'groovy-all'
-    }
+    compileOnly "org.grails:grails-core:$grailsVersion"
+
+    api "org.gebish:geb-spock:$gebSpock"
+    api "org.grails:grails-testing-support:$testingSupportVersion"
+    api "org.grails:grails-datastore-gorm:$datastoreVersion"
+
```

Comment:
I wonder if we should also add a comment on top of these dependencies to make it clear that these are there to be used transitively by the generated tests.

Reply:
I have added a comment.